services/stats/collector/collector.go

```diff
@@ -0,0 +1,152 @@
+package collector
```

Comment:
Use the same package i.e. stats. But call the file and the object runtime stat collector